pkgs/by-name/em/ember-cli/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, mkYarnPackage
+, fetchFromGitHub
+, fetchYarnDeps
+}:
+
+let
+  pname = "ember-cli";
+  version = "5.3.0";
+in
+mkYarnPackage rec {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "ember-cli";
+    repo = pname;
```

```suggestion
    repo = "ember-cli";
```
They are semantically different, this practice should be removed from nixpkgs.